rbd-target-api.py

```diff
@@ -688,6 +688,15 @@ def disk(image_id):
         logger.warning("disk create request failed: {}".format(msg))
         return jsonify(message=msg), 400
 
+    # Any disk operation must be started by the `local_gw` (owner)
+    # so `local_gw` must exist in the gateways configuration
+    if local_gw not in gateways:
+        msg = "local machine ({}) cannot be used to perform disk " \
+              "operations because it is not defined within gateways " \
+              "configuration".format(local_gw)
+        logger.warning("disk create request failed: {}".format(msg))
+        return jsonify(message=msg), 400
```

Comment:
It looks like this is a requirement for multiple operations. Check out the target, client, etc operations.

Move this to a new function so we can call in the other operations that need the same check.

Reply:
@mikechristie I've applied the validation to all operations where it's needed